Sources/NIOHPACK/HPACKHeader.swift

```diff
@@ -161,8 +161,27 @@ public struct HPACKHeaders: ExpressibleByDictionaryLiteral {
         
         return array
     }
+
+    /// Retrieves the first value for a given header field name from the block.
+    ///
+    /// This method uses case-insensitive comparisons for the header field name. It
+    /// does not return the first value from a maximally-decomposed list of the header fields,
+    /// but instead returns the first value from the original representation: that means
+    /// that a comma-separated header field list may contain more than one entry, some of
+    /// which contain commas and some do not. If you want a representation of the header fields
+    /// suitable for performing computation on, consider `getCanonicalForm`.
+    ///
+    /// - Parameter name: The header field name whose first value should be retrieved.
+    /// - Returns: The first value for the header field name.
+    public subscript(first name: String) -> String? {
```

Comment:
@weissi What are your thoughts about the spelling of this API? It sits a bit weirdly with me, but I don't know if it really matters.

Reply:
I think we should do something more similar to what the stdlib does for [`first(where:)`](https://developer.apple.com/documentation/swift/array/1848165-first). What about `first(name: String)`?

Reply:
That's fine by me: the only risk is that this type already has a `first(where:)` so that API may get lost in the autocorrect.

Reply:
`first(name:)` sounds good to me too.